.gitlab-ci.yml

```
@@ -247,15 +247,15 @@ regression-composer-works-behind-satellite-fallback:
   extends: .regression
   rules:
     # BLACKLIST: Skipped on subscribed RHEL machines
-    - if: $RUNNER !~ "/^.*(rhel-.*-ga|centos|fedora).*$/" && $RUNNER !~ "/^.*(rhel-9.5|rhel-10.0).*$/" && $CI_PIPELINE_SOURCE != "schedule"
+    - if: $RUNNER !~ "/^.*(rhel-.*-ga|centos|fedora).*$/" && $CI_PIPELINE_SOURCE != "schedule"
```

Is it easier and more readable to make it non-inverted?
```suggestion
    - if: $RUNNER ~ "/^.*(rhel-.*-nightly).*$/" && $CI_PIPELINE_SOURCE != "schedule"
```

---

Yeah sure that looks better, thanks!